client/packages/common/src/intl/locales/en/app.json

```diff
@@ -111,7 +115,8 @@
   "dispensary": "Dispensary",
   "programs": "Programs",
   "contact-trace": "Contact Tracing",
-  "prescription": "Prescriptions",
+  "prescription": "Prescription",
+  "prescriptions": "Prescriptions",
```

Comment:
Since we need both singular and plural forms for some of these, I've made the key and value consistent with each other, plural wise.